cds_ils/importer/providers/cds/rules/document.py

```diff
@@ -0,0 +1,948 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think I will skip to check in details this, but this mean that you can remove the rules from `cds-dojson`?
Honestly I like the idea of having this here.